glib/src/object.rs

```diff
@@ -1981,6 +1981,12 @@ pub trait ObjectExt: ObjectType {
     /// Downgrade this object to a weak reference.
     fn downgrade(&self) -> WeakRef<Self>;
 
+    // rustdoc-stripper-ignore-next
+    /// Downgrade this object to a local weak reference.
+    /// The clousure will be called when the the object is disposed.
+    #[doc(alias = "g_object_weak_ref")]
+    fn downgrade_local<F: Fn() + 'static>(&self, f: F) -> WeakRefLocal<Self, F>;
+
```

Comment:
Also I would call it something like `add_weak_notify()` or so

Reply:
changed the name.